appinventor/components/src/com/google/appinventor/components/common/YaVersion.java

```diff
@@ -588,7 +588,12 @@ private YaVersion() {
   // - WEB_COMPONENT_VERSION was incremented to 9
   // For YOUNG_ANDROID_VERSION 226:
   // - FORM_COMPONENT_VERSION was incremented to 31
-  public static final int YOUNG_ANDROID_VERSION = 226;
+  // For YOUNG_ANDROID_VERSION 227:
+  // - EMAILPICKER_COMPONENT_VERSION was incremented to 4
+  // - TEXTBOX_COMPONENT_VERSION was incremented to 7
+  // - PASSWORD_TEXTBOX_VERSION was incremented to 6
+
+  public static final int YOUNG_ANDROID_VERSION = 227;
```

Comment:
For future reference, we generally only update YOUNG_ANDROID_VERSION closer to a release since we may end up merging multiple PRs that would require a YOUNG_ANDROID_VERSION bump before a production release. This prevents creating merge conflicts on ucr.